### PR TITLE
[docs] - Graph-backed asset examples

### PR DESCRIPTION
**Pull request by clairelin135**
Updates graph-backed asset docs to add more examples about specifying asset dependencies

### CODE REVIEW

File: docs/content/concepts/assets/software-defined-assets.mdx [hunk 1]
```diff
@@ -236,6 +236,61 @@ graph_asset = AssetsDefinition.from_graph(store_slack_files_in_sql)
 
 **Note**: All output assets must be selected when using a graph-backed asset to create a job. Dagster will select all graph output automatically upon creating a job.
 
+#### Defining basic dependencies
+
+The `from_graph` attribute on the `AssetsDefinition` object will infer upstream and downstream asset dependencies from the graph definition provided. In the most simple case when the graph returns a singular output, Dagster infers the name of the graph to be the outputted asset key.
```
Comment (erinkcochran87) [inline docs/content/concepts/assets/software-defined-assets.mdx:241]:
```suggestion
The `from_graph` attribute on the `AssetsDefinition` object infers upstream and downstream asset dependencies from the graph definition provided. In the most simple case when the graph returns a singular output, Dagster infers the name of the graph to be the outputted asset key.
```

Comment (erinkcochran87) [reply docs/content/concepts/assets/software-defined-assets.mdx:241]: Suggesting we make this use active voice.

**Review by erinkcochran87 — COMMENTED** [review]
From a content perspective, this looks good to me.

**Review by yuhan — APPROVED** [review]
so good - i think it addressed all my confusion and i feel this content piece would be very helpful for folks to connect the dots between graphs and assets. 
thanks @clairelin135 for the quick turnaround!

**Pull request merged**